package.json

```diff
@@ -78,7 +78,7 @@
     "gl-mat4": "^1.2.0",
     "gl-mesh3d": "^2.0.0",
     "gl-plot2d": "^1.3.1",
-    "gl-plot3d": "^1.5.5",
+    "gl-plot3d": "git://github.com/archmoj/gl-plot3d.git#316309c4ede55f990a61df5ba91f0f52bad07374",
```

Comment:
https://github.com/gl-vis/gl-axes3d/compare/master...archmoj:rotate3d-axis-lables

Reply:
Today I worked on another version of gl-axes3d for better alignments (here: gl-vis/gl-axes3d@master...archmoj:rotate3d-axis-lables). Therefore the package.json files of gl-plot3d & plotly.js need second updates.